common/scripted_triggers/wc_trade_triggers.txt

```diff
@@ -0,0 +1,85 @@
+human_trade_trade_post_trigger = {
+	custom_tooltip = {
+		text = inHumantradepost
+		hidden_tooltip = {
+			OR = {
+				province_id = 269
+				province_id = 243			
+				province_id = 270
+				province_id = 324
+				province_id = 242
+				province_id = 315
+				province_id = 332
+				province_id = 339
+				province_id = 253
+				province_id = 252
+				province_id = 216
+				province_id = 198
+				province_id = 203
+				province_id = 3554
+				province_id = 3545
+				province_id = 3526
+				province_id = 266
+				province_id = 344
+				province_id = 345
+			}
+		}
+	}
+}
+
+azeroth_trade_trade_post_trigger = {
+	custom_tooltip = {
+		text = inAzerothiantradepost
+		hidden_tooltip = {
+			OR = {
+				province_id = 1
+				province_id = 14
+				province_id = 21			
+				province_id = 44
+				province_id = 17
+				province_id = 91
+				province_id = 50
+			}
+		}
+	}
+}
+
+available_to_build_trade_post_trigger = {
+	custom_tooltip = {
+		text = cannotBuildTradePost
+		hidden_tooltip = {
+			OWNER = { 
+				NOT = {
+					evil_government_trigger = yes

```

Comment:
There is no need in `evil_government_trigger = yes` if you have `evil_public_religion_trigger = yes`